### ISSUE TITLE
CLI: Make sure GET path '/' and '/api-docs' redirect to '/api-docs/'

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1159]
Or have some sort of simple index.html on '/' showing a link to `api-docs/`

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.1.7](https://github.com/open-wa/wa-automate-nodejs/compare/3.1.6...3.1.7) (2020-12-09)

- Release 3.1.7 [`d17d3f2`](https://github.com/open-wa/wa-automate-nodejs/commit/d17d3f2c19aa32fe60a5f9b9b25a614aa85bf2e5)
- fix: #1151 [`32b10c9`](https://github.com/open-wa/wa-automate-nodejs/commit/32b10c98b92703f3f55f7876f5d37a12f7f3aa19)
- Added WA_LICENSE_KEY as a env var for use when `inDocker` is `true` [`02ea436`](https://github.com/open-wa/wa-automate-nodejs/commit/02ea4366b6fe988c28c24595476aea5af2cd4739)
- `/` on CLI redirects to `/api-docs/` #1159 [`05dcbfc`](https://github.com/open-wa/wa-automate-nodejs/commit/05dcbfc13f45b298b44a0697127b37aa25912ac0)
- Docs: Fix sendText documentation link [`cd332d7`](https://github.com/open-wa/wa-automate-nodejs/commit/cd332d79867a54df9daa758a8a5c6636d87223de)
- patch: fix participant methods [`9359008`](https://github.com/open-wa/wa-automate-nodejs/commit/935900807a6d4f83fdcc0c013fde50670aab585a)
